docs/source/installation.md

```diff
@@ -228,7 +228,8 @@ Alternatively, to install all optional dependencies:
 ```bash
 git clone https://github.com/Project-MONAI/MONAI.git
 cd MONAI/
-pip install -e '.[all]'
+pip install -e '.[all]' (for Linux)
+pip install -e ".[all]" (for Linux/Windows)
```

Comment:
```suggestion
pip install -e ".[all]"
```

Thanks for spotting this. In Linux with bash the command with double quotes appears to work just fine (other shells may have issues) so I think we only need the one command.